shared-src/docs-developer-guide.rst

```diff
@@ -340,3 +340,87 @@ To exclude an built-in proselint check,
 specify the check name in the check list
 in the function :py:func:`exclude_checks`
 in the file :file:`style-test.py`.
+
+.. _docs-pdf:
+
+Building PDF for docs
+-----------------------
+
+The docs can be downloaded as PDF for the ease of users.
+
+To build the pdf locally, follow these steps:
+
+.. note::
+
+  All steps below explain the build process for odk1-docs. To build pdf for odk2-docs in a similar way, you can replace **1** with **2** in the above commands.
+
+1. Install XeLatex
```

Comment:
This only works for Ubuntu which might not be common. 
* Macs, https://www.tug.org/mactex
* Windows, https://miktex.org/

Reply:
Added reference to this!